stdlib/LinearAlgebra/src/factorization.jl

```diff
@@ -200,3 +180,27 @@ function ldiv!(Y::AbstractMatrix, A::Factorization, B::AbstractMatrix)
         return ldiv!(A, Y)
     end
 end
+
+function (/)(B::AbstractMatrix, F::Factorization)
+    require_one_based_indexing(B)
+    TFB = typeof(oneunit(eltype(B)) / oneunit(eltype(F)))
+    rdiv!(copy_similar(B, TFB), F)
+end
+# reinterpretation trick for complex lhs and real factorization
+function (/)(B::Union{Matrix{Complex{T}},Adjoint{Complex{T},Vector{Complex{T}}},Transpose{Complex{T},Vector{Complex{T}}}}, F::Factorization{T}) where {T<:BlasReal}
```

Comment:
Uh, this line is quite long. I'm not sure how best to wrap it though. The extreme other end would be
```suggestion
function (/)(
    B::Union{
        Matrix{Complex{T}},
        Adjoint{Complex{T},Vector{Complex{T}}},
        Transpose{Complex{T},Vector{Complex{T}}}
    },
    F::Factorization{T}
) where {T<:BlasReal}
```
Maybe there is some reasonable middle-ground?